changelog/2824.doc.rst

```diff
@@ -0,0 +1 @@
+Restructured sub-sections of the rot_a_to_b documentation to be consistent with other pages and added 'Raises', 'Example' sub-section.
```

Comment:
```suggestion
Restructured subsections of the `~plasmapy.formulary.mathematics.rot_a_to_b` documentation to be consistent with other pages, and added 'Raises' and 'Example' subsections.
```
📝 This suggestion will make it so that `rot_a_to_b` gets linked to from the online documentation.